src/core/interfaces/ISablierLockup.sol

```diff
@@ -52,6 +52,8 @@ interface ISablierLockup is ISablierLockupBase {
     /// @param transferable Boolean indicating whether the stream NFT is transferable or not.
     /// @param timestamps Struct encapsulating (i) the stream's start time and (ii) end time, all as Unix timestamps.
     /// @param cliffTime The Unix timestamp for the cliff period's end. A value of zero means there is no cliff.
+    /// @param unlockAmounts Struct encapsulating (i) the amount unlocked at the start time and (ii) the amount unlocked
```

Comment:
Should not be past tense here because `unlockAmounts` refer to the future.

```suggestion
    /// @param unlockAmounts Struct encapsulating (i) the amount to unlock at the start time and (ii) the amount to unlock
```

Reply:
correct